src/mscorlib/src/System/String.cs

```diff
@@ -1287,17 +1287,39 @@ internal String[] SplitInternal(char[] separator, int count, StringSplitOptions
 
             return result;
         }
-    
-    
-        // Removes a set of characters from the end of this string.
+       
+        // Removes single character from the Start or/and end of this string.
         [Pure]
-        public String Trim(params char[] trimChars) {
-            if (null==trimChars || trimChars.Length == 0) {
+        public String Trim(char c) {  
```

Comment:
Should the parameter be named `trimChar` to be consistent with the existing `trimChars`?